flit/__init__.py

```diff
@@ -116,8 +116,12 @@ def main(argv=None):
              )
     )
 
+    parser_publish.add_argument('--pypirc', default="~/.pypirc",
```

Comment:
I think we should handle it slightly differently if this is explicitly specified - it should check that the specified file exists, because if you've specified an exact path and it doesn't exist, that's probably a mistake.

That would mean not setting an explicit default here, so we can distinguish whether it's given at the command line or not.

Reply:
Thanks, that's a good suggestion! I removed the default so we can use None as a sentinel and checked the file existence. Because of that I needed to patch `os.path.isfile` in the unit tests. Let me know your thoughts, thanks!